graylog2-server/src/main/java/org/graylog2/lookup/adapters/CSVFileDataAdapter.java

```diff
@@ -191,6 +223,23 @@ protected void doRefresh(LookupCachePurge cachePurge) throws Exception {
         return newLookupBuilder.build();
     }
 
+    private void populateLookupRefs() throws IOException {
+
+        // populate lookup with CSV file
+        try (final InputStream inputStream = Files.newInputStream(Paths.get(config.path()));
```

Comment:
These two try-blocks could be extracted to a single method that takes the AtomicReference and a path.